lib/ansible/modules/system/iptables.py

```diff
@@ -226,6 +226,14 @@
         this, the source address is never altered."
     required: false
     default: null
+  syn:
+    version_added: "2.3"
+    description:
+      - "This allows matching packets that have the SYN bit set and the ACK
+        and RST bits unset. When negated, this matches all packets with
+        the RST or the ACK bits set."
+    required: false
```

Comment:
If it is not required, you don't have to add it. (The other parameters don't need `required: false` either)